nRF9160Master/src/Temp_MCP/TempAdcHandler.c

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file    Temp_Adchandler.c
+ * @author : Devendu
+ * @brief   File to calculate the temperature using MCP9700
+ * @date    09-04-2023
+ */
+/***********************************INCLUDES**************************/
+#include <nrfx_example.h>
+#include <saadc_examples_common.h>
+#include <nrfx_saadc.h>
+#include <zephyr/kernel.h>
+#include "TempAdcHandler.h"
+
+
+/************************************MACROS***************************/
+#define NRFX_LOG_MODULE                 EXAMPLE
+#define NRFX_EXAMPLE_CONFIG_LOG_ENABLED 1
+#define NRFX_EXAMPLE_CONFIG_LOG_LEVEL   3
+#include <nrfx_log.h>
```

Comment:
If we are using _printk_ for debug logs, Is it necessary to include this header here ? If we need it keep t there else remove it